zenpacklib.py

```diff
@@ -4997,8 +4997,9 @@ def format_message(e):
     def yaml_warning(loader, e):
         # Given a MarkedYAMLError exception, either log or raise
         # the error, depending on the 'fatal' argument.
-
-        print format_message(e)
+        pass
+        # commenting out for 1.1 release
+        # print format_message(e)
```

Comment:
Will we still be able to get the warning during development? 
Is there some develop flag that we can set?
Perhaps log.debug() ?